src/vivarium_public_health/risks/distributions.py

```diff
@@ -76,31 +79,43 @@ def initialization_requirements(self) -> Dict[str, List[str]]:
     def __init__(self, risk, weights, mean, sd):
         super().__init__()
         self.risk = EntityString(risk)
-        self._weights, self._parameters = self.get_parameters(weights, mean, sd)
+        self.weights = weights
```

Comment:
It's confusing to have attributes named both `weights` and `_weights`. Can you find better names for these new attributes? Maybe `_raw_weights` or `_input_weights`?